actionmailbox/test/unit/inbound_email_test.rb

```diff
@@ -12,6 +12,39 @@ class InboundEmailTest < ActiveSupport::TestCase
       assert_equal file_fixture("welcome.eml").read, create_inbound_email_from_fixture("welcome.eml").source
     end
 
+    test "multi-part mail can be built in tests using a block" do
+      inbound_email = create_inbound_email_from_mail do
+        to "test@example.com"
+        from "hello@example.com"
+
+        text_part do
+          body "Hello, world"
+        end
+
+        html_part do
+          body "<h1>Hello, world</h1>"
+        end
+      end
+
+      mail = inbound_email.mail
+
+      assert_equal mail.parts.length, 2
+      assert_equal mail.text_part.to_s, <<~TEXT.chomp
+        Content-Type: text/plain;\r
+         charset=UTF-8\r
+        Content-Transfer-Encoding: 7bit\r
+        \r
+        Hello, world
+      TEXT
+      assert_equal mail.html_part.to_s, <<~HTML.chomp
+        Content-Type: text/html;\r
+         charset=UTF-8\r
+        Content-Transfer-Encoding: 7bit\r
+        \r
+        <h1>Hello, world</h1>
+      HTML
+    end
```

Comment:
This test case is for `ActionMailbox::InboundEmail`. How about adding `ActionMailbox::TestHelperTest` in `test/unit/test_helper_test.rb`?

Reply:
Good point! I put it here since there wasn't an explicit test for `TestHelper`, but that makes a lot of sense to me.